policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/additions/tax_exempt_vt_interest_and_dividend_income.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class tax_exempt_vt_interest_and_dividend_income(Variable):
```

Comment:
```suggestion
class vt_tax_exempt_interest_and_dividend_income(Variable):
```

Reply:
rename .py and test files

Reply:
@PavelMakarchuk,  How does this name change resolve anything?  What feature of the VT income tax are you trying to model?

Reply:
This piece of the schedule IN-112
[2022 Schedule IN-112](https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf#page=1)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/d408a046-0a50-4f1b-a085-b1c2252f3809)
the name change was more cosmetic and does not address the larger question